ansible/site.yml

```
@@ -0,0 +1,377 @@
+    # Security tasks
+    - name: Reset UFW to defaults
+      ufw:
+        state: reset
+      when: enable_firewall | default(true)
+
+    - name: Set UFW default policies
+      ufw:
+        direction: "{{ item.direction }}"
+        policy: "{{ item.policy }}"
+      loop:
+        - { direction: 'incoming', policy: 'deny' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)